src/cascadia/TerminalSettingsModel/ActionArgs.h

```diff
@@ -286,6 +295,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         static constexpr std::string_view SuppressApplicationTitleKey{ "suppressApplicationTitle" };
         static constexpr std::string_view ColorSchemeKey{ "colorScheme" };
         static constexpr std::string_view ElevateKey{ "elevate" };
+        static constexpr std::string_view ContentKey{ "__content" };

```

Comment:
utterly horrified that somebody could put this in their JSON and do _something_ with it. What happens if they do? What if they use an existing content ID? what if they use a non-existent one?

Reply:
You know, I could have that raise a warning and throw it out I bet. The internal use of this ignore errors & warnings. But the settings from the user _could_ warn & discard the action....

Reply:
![image](https://user-images.githubusercontent.com/18356694/228307674-e7f45dc3-0026-49b4-a7c9-005e80952850.png)

and yea, it just worked